node/impl/full/state.go

```diff
@@ -1460,6 +1462,33 @@ func (m *StateModule) StateNetworkVersion(ctx context.Context, tsk types.TipSetK
 	return m.StateManager.GetNetworkVersion(ctx, ts.Height()), nil
 }
 
+func (a *StateAPI) StateActorCodeCIDs(ctx context.Context, nv network.Version) (map[string]cid.Cid, error) {
+	actorVersion, err := actors.VersionForNetwork(nv)
+	if err != nil {
+		return nil, xerrors.Errorf("invalid network version")
+	}
+
+	cids := make(map[string]cid.Cid)
+
+	manifestCid, ok := actors.GetManifest(actorVersion)
+	if !ok {
+		return nil, xerrors.Errorf("cannot get manifest CID")
+	}
+
+	cids["manifest"] = manifestCid
```

Comment:
```suggestion
	cids["_manifest"] = manifestCid
```
Just an option so that this will be displayed first in the JSON output, feel free to ignore.

Reply:
brilliant